src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -32,10 +34,7 @@
 import htsjdk.samtools.metrics.MetricBase;
 import htsjdk.samtools.metrics.MetricsFile;
 import htsjdk.samtools.metrics.StringHeader;
-import htsjdk.samtools.util.BlockCompressedOutputStream;
-import htsjdk.samtools.util.BlockCompressedStreamConstants;
-import htsjdk.samtools.util.IOUtil;
-import htsjdk.samtools.util.Log;
+import htsjdk.samtools.util.*;
```

Comment:
I don't know if we have a policy about this but I think we want to avoid `*` imports in our java code. There is an IDE setting to control this.